examples/pom.xml

```diff
@@ -49,7 +49,7 @@
         <dependency>
             <groupId>net.sourceforge.htmlunit</groupId>
             <artifactId>htmlunit</artifactId>
-            <version>2.15</version>
+            <version>2.36.0</version>
```

Comment:
Do we already have a CQ for this new version?

Reply:
Yes, a few of them are already processed and certified. For htmlunit it's this one:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21353

Reply:
Ok, great! Thanks for sharing the link.